### PR TITLE
[Doc] Improve GitHub issue template

### LINKED ISSUE #8878: Needing an FAQ
I am noticing a fair number of github issues which are being closed quite quickly, either because they are being identified as "questions", or because they are deemed to be not appropriate issues for github. I think an FAQ that could form part of the bitcoin core distrubution needs to be created so that github stops being used for out-of-scope issues - and the scope is documented, ideally along with links to where are the correct areas for asking questions. Raising this issue to hopefully kickstart that process.

### PR DESCRIPTION
**Pull request by fanquake**
Improve the layout/wording of the issue template, and link to the bitcoin StackExchange.
Possibly closes #8878

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,23 +1,35 @@
...
 
-### Any information in the debug.log file related to this issue?
-
-### Screenshots (if available)
+### Screenshots. 
+If the issue is related to the GUI, screenshots can be added to this issue via drag & drop.
 
-### What binary version was used (official or self compiled)
+### What version of bitcoin-core are you using?
+List the version number/commit ID, and if it is an official binary or self compiled.
```
Comment (laanwj) [inline .github/ISSUE_TEMPLATE.md]: maybe add 'or a distribution package such as PPA'

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,23 +1,35 @@
...
-### Machine specs: CPU, RAM, Disk space & OS (Windows, OS X, Linux) 
+### Machine specs:
+- OS: 
+- CPU: 
+- RAM: 
+- Disk size:
+- Disk Type (HD/SDD):
 
+### Any extra information that might be useful in the debugging process. 
+This is normally the contents of a  `debug.log` or `config.log` file. Raw text or a link to a pastebin type site are preferred.
```
Comment (MarcoFalke) [inline .github/ISSUE_TEMPLATE.md]: Maybe comment the comments out, so it is not required to delete them?

**Comment by MarcoFalke** [comment]
ACK

**Comment by paveljanik** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/8887/commits/0ce0b1b6b1449370e8c064898720fec30446aa67

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (fanquake) [inline .github/ISSUE_TEMPLATE.md]: Fixed.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (DIrkLeosBosch) [inline .github/ISSUE_TEMPLATE.md]: Hi, yes I find interesting, for me, is completely new territory.

Greeting´s 
427347_418593321560647_374744987_n


Dirk L. Feiler

**Pull request merged**